### PR TITLE
added menu page

### PR DESCRIPTION
**Pull request by Mr-Rovilay**
https://signin-signup-3.onrender.com

### CODE REVIEW

File: client/src/components/Offer.jsx [hunk 1]
```diff
@@ -3,25 +3,35 @@ import CountDown from "./CountDown";
...
+            Indulge in a culinary adventure with our delectable selection of
+            discounted delicacies, handcrafted to tantalize your taste buds.
+            Discover a world of flavors, where every bite tells a story of
+            passion and craftsmanship. Whether you crave the comforting warmth
+            of traditional favorites or the adventurous allure of bold,
+            innovative dishes, our menu has something to satisfy every palate.
+            Join us on a journey of culinary excellence, where each dish is a
+            celebration of quality ingredients and expert culinary artistry.
+            Elevate your dining experience today and savor the extraordinary at
+            our table
```
Comment (rovilay) [inline client/src/components/Offer.jsx:22]: Put this text in a dark transparent background or remove it.

File: client/src/pages/MenuPage.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+              >
+                <Link to={item.slug}>
+                  <div className="p-5 flex flex-col items-center text-center">
+                    <div className="rounded-xl overflow-hidden">
+                      <img src={item.img} alt={item.title} />{" "}
+                    </div>
+                    <h5 className="text-2xl md:text-3xl font-medium mt-3">
+                      {item.title}
+                    </h5>
+                    <p className="text-dark-grey text-2xl mt-3">{item.desc}</p>
```
Comment (rovilay) [inline client/src/pages/MenuPage.jsx:24]: your menu page isn't properly designed.

File: client/src/pages/MenuPage.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+                    <button className="text-center text-white text-2xl bg-green p-4 py-3 rounded-lg font-semibold mt-4 hover:bg-dark-green focus:scale-95 transition-all duration-200 ease-out">
+                      Click here to Explore More
+                    </button>
```
Comment (rovilay) [inline client/src/pages/MenuPage.jsx:27]: Your buttons should have the same styles all through the app. Compare `add to cart` and `signup`  buttons.

**Review by rovilay — CHANGES_REQUESTED** [review]

File: client/src/pages/MenuPage.jsx (hunk 2 above)
Comment (Mr-Rovilay) [reply client/src/pages/MenuPage.jsx:25]: check the review bro.. im done with this

**Pull request merged**